editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1839,17 +1839,17 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 
 					clicked = ObjectID();
 
-					if ((spatial_editor->get_tool_mode() == Node3DEditor::TOOL_MODE_SELECT && b->is_command_or_control_pressed()) || spatial_editor->get_tool_mode() == Node3DEditor::TOOL_MODE_ROTATE) {
+					if (can_select_gizmos && ((spatial_editor->get_tool_mode() == Node3DEditor::TOOL_MODE_SELECT && b->is_command_or_control_pressed()) || spatial_editor->get_tool_mode() == Node3DEditor::TOOL_MODE_ROTATE)) {
```

Comment:
What is this change for?

Reply:
The code below that section is where selection the selection process for 3D takes place. The modified code adds a check to make sure that a gizmo can be selected (implying something was selected, and one exists) before entering the body of the if statements, where transforms begin, and the encompassing case is broken from, preventing selection.

Reply:
Ah, for some reason I thought this already exists in 3D and the PR is 2D only 🙃

Reply:
All good. I could have separated the PR but I figured it would by jarring if 2D did one thing and 3D did another.